core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -149,8 +149,22 @@ endif
 
 CFG_DRAM_BASE    ?= 0xc0000000
 CFG_DRAM_SIZE    ?= 0x40000000
-CFG_STM32MP1_SCMI_SHM_BASE ?= 0x2ffff000
-CFG_STM32MP1_SCMI_SHM_SIZE ?= 0x00001000
+
+# CFG_STM32MP1_SCMI_SHM_BASE and CFG_STM32MP1_SCMI_SHM_SIZE, when not 0,
```

Comment:
CFG_STM32MP1_SCMI_SHM_SIZE will always be 0x1000 with this patch

Reply:
yes. I'll fix the description.